lib/model/content.dart

```diff
@@ -1022,18 +1022,28 @@ class _ZulipContentParser {
           }
           span = CodeBlockSpanNode(text: text, type: CodeBlockSpanType.text);
 
-        case dom.Element(localName: 'span', :final text, :final className):
-          final CodeBlockSpanType type = codeBlockSpanTypeFromClassName(className);
-          switch (type) {
-            case CodeBlockSpanType.unknown:
+        case dom.Element(localName: 'span', :final text, :final classes):
+          // The general form for Pygments nodes with multiple classes observed
+          // are, that the first class is the standard token class and trailing
+          // tokens are non-standard tokens. Zulip web client only styles the
+          // standard tokens classes. Same is done here, only standard token
+          // class names are emitted.
```

Comment:
```suggestion
          // Empirically, when a Pygments node has multiple classes, the first
          // class names a standard token type and the rest are for non-standard
          // token types specific to the language.  Zulip web only styles the
          // standard token classes and ignores the others, so we do the same.
```

The "are emitted" confused me: we don't emit any class names here, right? We consume class names, and may emit a CodeBlockSpanType value.